style.css

```diff
@@ -87,3 +88,212 @@ body {
   align-items: baseline;
   justify-content: flex-start;
 }
+
+.projects-section {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  gap: 12px;
+  margin-top: 650px;
+  width: 327px;
+  height: 552px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.cards {
+  width: 295px;
+  height: 220px;
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
```

Comment:
This is a nice card 👍🏽 but it's not centered yet, make sure to center it so that it matches the design. See the differences 👇🏽 

## Figma
![Screenshot 2023-02-17 at 21 29 31](https://user-images.githubusercontent.com/45739963/219827063-bf199e11-9301-4c34-8d50-60c5467d652f.png)

## Your design
![Screenshot 2023-02-17 at 21 29 54](https://user-images.githubusercontent.com/45739963/219827075-2cd6f05e-7e4f-4653-9c7f-a66e8b6af13b.png)